junebug/tests/test_api.py

```diff
@@ -103,20 +103,16 @@ def test_http_error(self):
     @inlineCallbacks
     def test_redirect_http_error(self):
         resp = yield self.get('/channels')
-        [redirect] = resp.history()
         yield self.assert_response(
-            redirect, http.MOVED_PERMANENTLY,
+            resp, 308,
             None, {
                 'errors': [{
-                    'code': 301,
-                    'message': '301 Moved Permanently: None',
+                    'code': 308,
+                    'message': '308 Permanent Redirect: None',
                     'new_url': '%s/channels/' % self.url,
-                    'type': 'Moved Permanently',
+                    'type': 'Permanent Redirect',
                 }],
             })
-        yield self.assert_response(
-            resp, http.OK,
-            'channels listed', [])
```

Comment:
Is there a reason for removing this assert? This makes sure that the URL we redirected to works.

Reply:
It doesn't actually do the redirect, `[redirect] = resp.history()` failed so I had to remove it. 

The first assert is now on `resp` because `redirect` doesn't exist. @rudigiesler

Reply:
Ah, it must be the client that doesn't understand 308 now. 👍